pkg/operator/operator.go

```diff
@@ -206,6 +207,11 @@ func (optr *Operator) maoConfigFromInfrastructure() (*OperatorConfig, error) {
 		return nil, err
 	}
 
+	baremetalImages, err := getAdditionalProviderImages(provider, *images)
```

Comment:
This should be refactored.  We should check if we need to do this prior to getting the images rather than putting a bunch of empty images in the operator.

Reply:
The image struct is populated by this time. I agree that we don't have to put empty images.